Scripts/STRAXSidechainRegistrationScript.ps1

```diff
@@ -0,0 +1,593 @@
+﻿#Create Functions
+function Get-IndexerStatus
+{
+    $Headers = @{}
+    $Headers.Add("Accept","application/json")
+    $AsyncLoopStats = Invoke-WebRequest -Uri http://localhost:$API/api/Dashboard/AsyncLoopsStats | Select-Object -ExpandProperty content 
+    if ( $AsyncLoopStats.Contains("Fault Reason: Missing outpoint data") )
+    {
+        Write-Host "ERROR: Indexing Database is corrupt" -ForegroundColor Red
+        Write-Host "Would you like to delete the database?"
+        $DeleteDB = Read-Host -Prompt "Enter 'Yes' to remove the Indexing Database or 'No' to exit the script"
+        While ( $DeleteDB -ne "Yes" -and $DeleteDB -ne "No" )
+        {
+            $DeleteDB = Read-Host -Prompt "Enter 'Yes' to remove the indexing database or 'No' to exit the script"
+        }
+        Switch ( $DeleteDB )
+        {
+            Yes 
+            {
+                Shutdown-MainchainNode
+                Remove-Item -Path $API\addressindex.litedb -Force
+                if ( -not ( Get-Item -Path $API\addressindex.litedb ) )
+                {
+                    Write-Host "SUCCESS: Indexing Database has been removed. Please re-run the script" -ForegroundColor Green
+                    Start-Sleep 10
+                    Exit
+                }
+                    Else
+                    {
+                        Write-Host "ERROR: Something went wrong. Please contact support in Discord" -ForegroundColor Red
```

Comment:
Probably should have an `exit` statement in case of corrupt DB and failure to remove the litedb file
```suggestion
                        Write-Host "ERROR: Something went wrong. Please contact support in Discord" -ForegroundColor Red
                        Exit
```